Package.resolved

```diff
@@ -67,4 +67,4 @@
     ]
   },
   "version": 1
-}
+}
```

Comment:
add newline

Reply:
This is automatically generated by SwiftPM, not hand-made